xrayvision/Visibility.py

```diff
@@ -47,9 +55,59 @@ def from_map(self, inmap):
         """
         return Transform.dft(inmap, self.uv, self.vis)
 
-    def to_map(self, outmap):
+    def from_map_v2(self, inmap, center=None, pixel_size=None):
+        """
+        Parameters
+        ----------
+        inmap: np.array
+            The input image
+        center: array-like
+            Where should be the origin in the Fourier-transformation
+        pixel_size: array-like
+            What is the size of one pixel in the given direction (x, y)
+        Returns
+        -------
+
+        """
+        if not center:
+            center = self.xyoffset
```

Comment:
If we are converting from a 'map' it will have implicit centre this would be the place to set it. I think here might be a better place for the default values `xyoffset=(0., 0.), pixel_size=(1., 1.)`